simulator/src/test/java/com/hazelcast/simulator/coordinator/CoordinatorRunTestSuiteTest.java

```diff
@@ -0,0 +1,88 @@
+package com.hazelcast.simulator.coordinator;
+
+import com.hazelcast.simulator.agent.workerjvm.WorkerJvmSettings;
+import com.hazelcast.simulator.coordinator.remoting.AgentsClient;
+import com.hazelcast.simulator.test.TestCase;
+import com.hazelcast.simulator.test.TestSuite;
+import com.hazelcast.simulator.worker.commands.Command;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class CoordinatorRunTestSuiteTest {
+
+    private final TestSuite testSuite = new TestSuite();
+
+    @Mock
+    private final WorkerJvmSettings workerJvmSettings = new WorkerJvmSettings();
+
+    @Mock
+    private final AgentsClient agentsClient = mock(AgentsClient.class);
+
+    @InjectMocks
+    private Coordinator coordinator;
+
+    @Before
+    public void setUp() {
+        System.setProperty("user.dir", "./dist/src/main/dist");
```

Comment:
Hmm, I can't import the project under Windows ATM. The whole `tests` module has some problems with the `pom.xml` :(

But it should be safe to use `/` on all platforms, since Java will deal with it. http://stackoverflow.com/a/2417546